page-optimize.php

```diff
@@ -36,10 +36,16 @@ function page_optimize_cache_cleanup( $file_age = DAY_IN_SECONDS ) {
 		return;
```

Comment:
I think we will always hit this early return when caching is disabled with an empty `PAGE_OPTIMIZE_CACHE_DIR`.